### ISSUE TITLE
Shared tooltip not working on radar and bar charts

### ISSUE THREAD

**Issue by bearomera** [apexcharts/apexcharts.js#1575]
The option of shared tooltip is not working in radar and bar charts.

In the radar chart when entering that option, the tooltip just doesn't appear:
https://codepen.io/bromera/pen/vYNJqjE

In the bar chart, the tooltip appears but without applying the shared settings:
https://codepen.io/bromera/pen/Baomvaz

**Comment by timtilch** [comment]
I have the same problem

**Comment by sharq1** [comment]
+1

**Comment by sharq1** [comment]
When I enable `shared` I get an error that it can't be set at the same with `intersect: true`.
When I set both properly, the tooltip does not appear at all.

```js
shared: true,
intersect: false,
```

**Comment by pmsoltani** [comment]
Not stale

**Comment by pmsoltani** [comment]
Not stale

**Comment by Splinter0** [comment]
Any progress on this?

**Comment by JshGrn** [comment]
not stale/completed/still a problem

**Comment by victorvargasr** [comment]
Try something like this and work on it, I found it useful: 

`tooltip: {
  enabled: true,
  style: {
    fontSize: "12px"
  },
  y: {
    formatter: function(value, {series, seriesIndex, dataPointIndex, w}) {
      return "%" + value;
    }
  },
  x: {
    formatter: function( value, { series, seriesIndex, dataPointIndex, w } ) {
      return value;
    }
  }
}`

**Comment by victorvargasr** [comment]
Try something like this and work on it, I found it useful: 

`tooltip: {
  enabled: true,
  style: {
    fontSize: "12px"
  },
  y: {
    formatter: function(value, {series, seriesIndex, dataPointIndex, w}) {
      return "%" + value;
    }
  },
  x: {
    formatter: function( value, { series, seriesIndex, dataPointIndex, w } ) {
      return value;
    }
  }
}`

**Comment by Victorm0612** [comment]
Try something like this and work on it, I found it useful for radar-type charts: 

`tooltip: {
  enabled: true,
  style: {
    fontSize: "12px"
  },
  y: {
    formatter: function(value, {series, seriesIndex, dataPointIndex, w}) {
      return "%" + value;
    }
  },
  x: {
    formatter: function( value, { series, seriesIndex, dataPointIndex, w } ) {
      return value;
    }
  }
}`

**Comment by FernandoBonfimArctica** [comment]
same problem here

**Comment by junedchhipa** [comment]
Fixed and released in **6.10.0** (`ddb97d2da`). Radar half first, then the bar half, then one limitation that remains.

**Radar.** Verified on 6.10.0, two series, hovering a data point with `tooltip: { shared: true }`:

```
before: no tooltip at all
now:    visible, 2 rows, values ["40", "25"]
```

The cause is worth writing down because it explains why it looked so arbitrary. Radar is the only chart that is hit-tested through its markers while *not* being an xyChart: `Core` puts it in `axisChartsArrTypes` but not in `xyChartsArrTypes`. In `Tooltip.addSVGEvents` that meant it matched none of the three wiring branches once `showOnIntersect` was false, so it got **no hover listeners whatsoever**. And `tooltip.shared: true` lands exactly there, because the config validator rejects `shared` and `intersect` together, so asking for a shared tooltip forces `intersect: false` and silently takes the listeners away with it. That is why `shared: true` specifically appeared to disable the tooltip.

A second, independent failure was fixed alongside it: `markers.size: 0` left a 0x0 marker with nothing to hover, so there was no tooltip in *any* mode. Our own `radar-multiple-series` demo was configured that way. A hidden marker now keeps an invisible hit area, painted transparent so it still receives pointer events while painting nothing.

**Bar.** The bar half of this report was already fixed before 6.10.0: a shared tooltip on a stacked or grouped bar chart renders one row per series (checked at 2 of 2).

**Still true:** a radar cannot be *swept* the way a line or area chart can. It has no plot-area hover with polar nearest-point resolution, so the pointer has to pass over a data point. Every tooltip mode works when it does. If sweeping a radar matters to you, that is worth its own issue.

Guarded by `tests/interaction/specs/radar-tooltip.spec.js`, which is the first radar interaction spec in the repo. There were none, which is a fair part of why this survived six years: the visual suite only screenshots, so it never hovered anything.

Apologies for the six years of silence on this one. Thanks to everyone who kept it alive with repros.
